blueprints/frontend.py

```diff
@@ -307,7 +307,7 @@ async def profile_select(id):
         'SELECT name, safe_name, id, priv, country '
         'FROM users '
         'WHERE safe_name IN (%s) OR id IN (%s) LIMIT 1',
```

Comment:
why is this using `IN`? am i missing something?
```suggestion
        'WHERE safe_name = %s OR id = %s LIMIT 1',
```

Reply:
ur right, this code is dum lol